tests/integration/test_charm.py

```diff
@@ -144,7 +176,7 @@ async def test_with_ingress(
     # mypy doesn't see that ActiveStatus has a name
     await model.wait_for_idle(status=ActiveStatus.name)  # type: ignore
 
-    traefik_ip = next(await get_unit_ips(traefik_app_name))
+    traefik_ip = next(iter(await get_unit_ips(traefik_app_name)))
```

Comment:
Out of curiosity, what is the reason for needing the `iter` here?

Reply:
It's due to that `get_unit_ips` returns a tuple, I think I will change this to `(await get_unit_ips(traefik_app_name)[0]`, thanks!